docker-maven-plugin/src/main/java/org/openmrs/maven/plugins/AbstractDockerMojo.java

```diff
@@ -21,10 +21,10 @@
 
 abstract class AbstractDockerMojo extends AbstractMojo {
 
-    protected static final String DEFAULT_MYSQL_CONTAINER = "openmrs-sdk-mysql-v3-2";
+    protected static final String DEFAULT_MYSQL_CONTAINER = "openmrs-sdk-mysql-v3-3";
     protected static final String DEFAULT_MYSQL_EXPOSED_PORT = "3308";
     protected static final String DEFAULT_MYSQL_PASSWORD = "Admin123";
-    protected static final String MYSQL_5_6 = "mysql:5.6";
+    protected static final String MYSQL_5_6 = "mysql:8.4.1";
```

Comment:
Is there a reason why you still name the constant as 5_6?

Reply:
I renamed the variable.